REQUIRE

```diff
@@ -1,5 +1,5 @@
 julia 0.5
-DiffBase 0.0.3 0.3.0
+DiffBase 0.0.3 0.4.0
```

Comment:
looks like this loosening won't matter until there's an allowed ForwardDiff version that isn't upper-bounding DiffBase to < 0.3